mappings/net/minecraft/util/CsvOutput.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_4456 net/minecraft/util/CsvOutput
+	CLASS class_4457 Header
+		FIELD field_20286 columns Ljava/util/List;
+		METHOD method_21631 startBody (Ljava/io/Writer;)Lnet/minecraft/class_4456;
+		METHOD method_21632 column (Ljava/lang/String;)Lnet/minecraft/class_4456$class_4457;
```

Comment:
This is a method. Shouldn't it be something like `addColumn`?

Reply:
I originally named this like a builder. It worthes some debate on whether this should be as is or be named as a builder.